pallets/ajuna-affiliates/src/benchmarking.rs

```diff
@@ -136,21 +123,8 @@ benchmarks_instance_pallet! {
 	}
 
 	impl_benchmark_test_suite!(
-		Affiliates,
-		new_test_ext(),
-		Test
+		Pallet,
+		crate::mock::new_test_ext(),
+		crate::mock::Test
 	);
 }
-
-pub fn new_test_ext() -> sp_io::TestExternalities {
-	let t = frame_system::GenesisConfig::<Test>::default().build_storage().unwrap();
-	let mut ext = sp_io::TestExternalities::new(t);
-	ext.execute_with(|| System::set_block_number(1));
-	ext.execute_with(|| {
-		let acc_1 = account::<Test, ()>(ACC_1);
-		MockAccountManager::set_organizer(acc_1);
-		MockAccountManager::try_add_to_whitelist(&AffiliateWhitelistKey::get(), &acc_1)
-			.expect("Should add to whitelist");
-	});
-	ext
-}
```

Comment:
The setup must not contain anything besides the default externalities. All custom state must be done in the benchmark setup function.